templates/_partials/fields/image.html.twig

```diff
@@ -6,7 +6,7 @@
     <editor-image
         :name="'{{ name }}'"
         :filename="'{{ field.get('filename') }}'"
-        :thumbnail="'{{ field|thumbnail(width=1200, height=1200, area='files', path=setPath) }}'"
+        :thumbnail="'{{ field|thumbnail(width=1200, height=1200, area='files') }}'"
```

Comment:
area param does nothing during thumbnail generation, is this something you plan to add later?

Reply:
Maybe, but if we do, i can put it back.. removed for now.